new-chatbot-app/client/src/components/OfflineTicketWidget.jsx

```diff
@@ -9,7 +9,7 @@ const OfflineTicketWidget = () => {
   const [name, setName] = useState('');
   const [details, setDetails] = useState('');
 
-  const { socketContextValues: scv } = useContext(SocketContext);
+  const { socketContextValues: socketContextValues } = useContext(SocketContext);
```

Comment:
If you use type socketContextValues, I recommend changing the context variable to a different name (maybe a character change) for best convention

Reply:
Yeah I was supposed to remove that, I must have missed that one while I was going over the code. Will fix it in the next PR.